miniapps/tools/CMakeLists.txt

```diff
@@ -53,16 +53,22 @@ add_mfem_miniapp(tmop-metric-magnitude
   ${MFEM_MINIAPPS_COMMON_HEADERS}
   LIBRARIES mfem-common)
 
+add_mfem_miniapp(least-squares
+  MAIN least-squares.cpp LIBRARIES mfem)
```

Comment:
Will `least-squares` work if `MFEM_USE_MPI=OFF`?  I suspect there may be issues because the `Par`-prefixed classes are used, but I could be wrong